### ISSUE TITLE
Unable to Run a query in Massmine

### ISSUE THREAD

**Issue by AnushaVanama** [JoshCMoore/Massmine-for-the-Masses#224]
I tried creating a new account using twitter API keys.
Received an error after submitting a query using search term Keyword and count.

@JoshCMoore @PatriciaTanzer  @LoganHornbuckle  @Momac546  - Kindly find the attached error 
[error.txt](https://github.com/JoshCMoore/Massmine-for-the-Masses/files/3636457/error.txt)
and please let me know how to debug this issue
![errorscreenshot](https://user-images.githubusercontent.com/49127881/65336868-1d517b00-db95-11e9-8d80-0b5ba54c38fc.png)

[ExceptionPexpect at _query_make_query_.zip](https://github.com/JoshCMoore/Massmine-for-the-Masses/files/3636516/ExceptionPexpect.at._query_make_query_.zip)

**Comment by Momac546** [comment]
Do you have massmine installed? If not follow the instructions on massmines
website to get it installed. You can find them here:http://www.massmine.org/
.

Let me know how it goes.

**Comment by AnushaVanama** [comment]
@Momac546 

Yes, I had installed massmine.

I had downloaded it from http://www.massmine.org/docs/install.html and executed below 2 commands as mentioned in the user manual documentation

• unzip django/webapp/massmine-1.1.0-linux-x86_64.zip -d django/ 
• cd django/massmine && sudo ln -s `pwd`/massmine/usr/local/bin

Kindly let me know if I need to do anything apart from this

**Comment by Momac546** [comment]
Can you use the massmine commands in the command line without errors?

**Comment by AnushaVanama** [comment]
@Momac546  Thanks for your help. Now I am able to run the query successfully.
I had gone through those commands again and found that the symbolic link which I had created previously is incorrect.

In the User manual, there is no space between massmine and /usr. But in the http://www.massmine.org/docs/install.html link, I found that there is a space between massmine and /usr.

sudo ln -s `pwd`/massmine  /usr/local/bin


![Successful query](https://user-images.githubusercontent.com/49127881/65351454-26067900-dbb6-11e9-9500-831d61921f37.png)
